Jetstream/Session.swift

```diff
@@ -33,14 +33,13 @@ public class Session {
     let client: Client
     var nextMessageIndex: UInt = 1
     var serverIndex: UInt = 0
-    var scopes = [UInt: Scope]()
+    var scopes = [UInt: (Scope, [String: AnyObject])]()
```

Comment:
Sadly this is not working, I've tried a multitude of different ways even using Typealiases.  It keeps saying "Expected ',' separator.  Invalid character in source file":

```swift
typealias ScopeWithFetchParams = (scope: Scope, fetchParams: [String: AnyObject])
// ..
    var scopes = [UInt: ScopeWithFetchParams]()
```